### PR TITLE
mark the new features as PREVIEW in description 

### PR DESCRIPTION
**Pull request by xizha162**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json [hunk 1]
```diff
@@ -1165,7 +1165,7 @@
         "name": "AgentPoolType",
         "modelAsString": true
       },
-      "description": "AgentPoolType represents types of an agent pool"
+      "description": "AgentPoolType represents types of an agent pool. VirtualMachineScaleSets type is still in PREVIEW."
```
Comment (jluk) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json]: The correlation between agentpooltype and vmss isn't super clear - is there somewhere we state type = vmss is a pre-requisite for agentpool? We'd probably want to state it for AZ as well.

Comment (xizhamsft) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json]: marking vmss as required for az. For multi-agentpool, will include the change when it is ready in the next release.

**Comment by nschonni** [comment]
If the API is in preview, should it be under the preview folder?

**Comment by jluk** [comment]
@nschonni the preview descriptors are strictly talking about those properties, the api itself actually is not a preview api version. we have these preview properties exposed because AKS is using a feature opt-in flag for customers so it is blatantly clear to them when they opt-in to something very fresh and it requires explicit opt-in to use those preview properties.

**Review by jluk — APPROVED** [review]
LGTM

**Comment by anuchandy** [comment]
synced with @xizhamsft, though this property is preview it just mean that functionality behind it is in preview but it won't be removed or changed in a breaking way.

**Review by anuchandy — APPROVED** [review]

**Pull request merged**